### PR TITLE
OCI GG script to clean up trails

### PR DESCRIPTION
**Pull request by eloilopes**
New script to clean up OCI GG trail files.

### CODE REVIEW

**Review by AlexanderHodicke — COMMENTED** [review]
@oheimburger Can you review the code please

File: data-platform/data-integration/oci-goldengate/scripts/clean-up-ocigg-trail-files.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+from base64 import b64encode
+import base64
+
+# Auth Config
+CONFIG_PROFILE = "DEFAULT"
+config = oci.config.from_file('~/.oci/config', CONFIG_PROFILE)
+
+#OCI GoldenGate API
+version ="v2"
+username = '<gg username>'
```
Comment (oheimburger) [inline data-platform/data-integration/oci-goldengate/scripts/clean-up-ocigg-trail-files.py:52]: Can this be a configurable item outside of this script (like a configuration file, wallet, call parameter etc.)?

**Review by oheimburger — CHANGES_REQUESTED** [review]

File: data-platform/data-integration/oci-goldengate/scripts/clean-up-ocigg-trail-files.py (hunk 1 above)
Comment (eloilopes) [reply data-platform/data-integration/oci-goldengate/scripts/clean-up-ocigg-trail-files.py:52]: I've never tried. All documentation examples show in that way.

Comment (oheimburger) [reply data-platform/data-integration/oci-goldengate/scripts/clean-up-ocigg-trail-files.py:52]: Standard Python features may be used https://docs.python.org/3/library/argparse.html#module-argparse

**Pull request closed without merging**